docs/guide/catalog/set_up_products.md

```diff
@@ -0,0 +1,52 @@
+# Set up product for purchasing
```

Comment:
For doc reviewers: not quite satisfied with the title, happy to hear better alternatives